README.md

```diff
@@ -47,9 +47,14 @@ variations.
 
 Each test case has the the following keys:
 - description: which will be used to name each generated test
+  - the description should not simply explain **what** each case is (that is redundant information)
+  - the description should explain **why** each case is there
+  - for example, what kinds of incorrect implementations might this case help us find?
 - 'variable names': one or more variable names with values which will be passed to the solution method
```

Comment:
so I guess here "solution method" means "method in the solution". In leap, we'd get `"year":1996` telling us to do `is_leap 1996` or `isLeap(1996)` or similar. I do think "solution method" makes more sense than "test method". Basically I think it's to make clear that it's what the student wrote and is submitting.

If it still bears improvement, maybe I could offer "function under test", or simply "the submitted function" (maybe too specific, if not all submissions are single functions). I would have also offered "the student's function" but that canonicalises the student word that I use, whereas other people use others (practitioner, exercist, user...)